PubSub/tests/System/ManageSubscriptionsTest.php

```diff
@@ -218,6 +218,49 @@ public function testSubscriptionDurations($client)
         $this->assertEquals($durationNanos, $info['expirationPolicy']['ttl']['nanos']);
     }
 
+    /**
+     * @dataProvider clientProvider
+     */
+    public function testDeadLetterPolicy($client)
+    {
+        if ($client instanceof PubSubClientRest) {
+            $this->markTestSkipped(
+                'deadLetterPolicy not available in REST transport during experimental period.'
```

Comment:
This is when DLQ is alpha?

Reply:
I'm unsure when the new feature will be published to the REST definition. We've had the same issue with ordering keys.

Reply:
For now, it's likely we're going to expose these when the server-side goes GA.

Reply:
Has this been actually resolved?

Reply:
The latest version of the discovery document does not include dead letter queues: https://pubsub.googleapis.com/$discovery/rest?version=v1

Reply:
Should we notate this in the client?